bleach/sanitizer.py

```diff
@@ -668,8 +668,20 @@ def disallowed_token(self, token):
             assert token_type in ("StartTag", "EmptyTag")
             attrs = []
             for (ns, name), v in token["data"].items():
+                # If we end up with a namespace, but no name, switch them so we
+                # have a valid name to use.
+                if ns and not name:
```

Comment:
This is such a weird case. I don't see why the tokenizer should ever parse an html attribute as having a namespace, but no name. However, it's harder to fix this in html5lib than it is to just swap the values. So I swapped the values.